srml/contract/src/rent.rs

```diff
@@ -110,14 +129,15 @@ fn try_evict_or_and_pay_rent<T: Trait>(
 					.rent_allowance -= imbalance.peek(); // rent_allowance is not exceeded
 			})
 		}
-		return false;
+
+		RentOutcome::Paid
```

Comment:
well kind of because it is not very paid in case `pay_rent=false`

Reply:
Yeah, kinda bother me as well!